doc/source/overview/roadmap.md

```diff
@@ -19,7 +19,6 @@ It will also be possible to combine different algorithms in ensemble detectors.
 The library **currently** covers both online and offline **outlier** detection algorithms for 
 tabular data, images and time series as well as offline **adversarial** detectors for 
 tabular data and images. Current **drift** detection capabilities cover almost any data modality such as mixed type tabular data, 
-text, images or graphs both in the online and offline setting, and with .
+text, images or graphs, both in the online and offline setting. Furthermore, Alibi Detect provides supervised (*concept*) drift and context-aware drift detectors.
```

Comment:
I think we should remain careful when it comes to describing supervised drift detectors as 'concept' drift detectors.

Reply:
True, thought I would add naming that most users might be more familiar with, but happy to take it out if we want to be a bit cautious with throwing the term concept drift around!

Reply:
@arnaudvl @ojcobb did we decide on this?

Reply:
I think given that we define concept drift to be something specific in our 'what is drift' docs it is a bit confusing if we use it to mean other things as well.